jsonnet/jsonnetfile.lock.json

```diff
@@ -4,7 +4,7 @@
     {
       "source": {
         "git": {
-          "remote": "https://github.com/brancz/kubernetes-grafana",
```

Comment:
Can you not commit the entire jsonnetfile.lock.json, just the ones related to etcd. We will bump everything in the next few weeks. 

Can you just commit the lines around etcd and then regenerate again, thanks! :)

Reply:
If I remove everything but the etcd changes in the lock file and then re-run `make generate-in-docker` I get no updates to the rules, and if I run `jb update` I get all the changes you see in this PR — what is the procedure for updating just etcd?